buildenv/jenkins/docker-slaves/jenkinsfile-build-container

```diff
@@ -24,6 +24,10 @@
 def ARCH = (params.ARCH) ? params.ARCH : "unknown"
 def OS = (params.OS) ? params.OS : "unknown"
 def validArch = ["x86", "s390x", "ppc64le"]
+def NAMESPACE = (params.NAMESPACE) ? params.NAMESPACE : "eclipse"
+def REPOSITORY = (params.REPOSITORY) ? params.REPOSITORY : "${NAMESPACE}/openj9-jenkins-agent-${ARCH}-${OS}"
```

Comment:
I think this will have to move below the `if` statement below since the OS/ARCH will be "unknown" at this point for PR builds.